bot/exts/evergreen/wolfram.py

```diff
@@ -62,7 +62,8 @@ async def predicate(ctx: Context) -> bool:
             # if the invoked command is help we don't want to increase the ratelimits since it's not actually
             # invoking the command/making a request, so instead just check if the user/guild are on cooldown.
             guild_cooldown = not guildcd.get_bucket(ctx.message).get_tokens() == 0  # if guild is on cooldown
-            if not any(r.id in ignore for r in ctx.author.roles):  # check user bucket if user is not ignored
+            # check the message is in a guild, and check user bucket if user is not ignored
+            if ctx.guild and not any(r.id in ignore for r in ctx.author.roles):
```

Comment:
This should definitly have send a failure message.

Reply:
This change is purely just for when wolfram is being invoked from the help command. If a user actually calls `.wa` it will raise an error.